execution/gethexec/executionengine.go

```diff
@@ -80,9 +80,10 @@ type ExecutionEngine struct {
 	resequenceChan    chan []*arbostypes.MessageWithMetadata
 	createBlocksMutex sync.Mutex
 
-	newBlockNotifier chan struct{}
-	latestBlockMutex sync.Mutex
-	latestBlock      *types.Block
+	newBlockNotifier  chan struct{}
+	reorgEventsReader chan struct{}
```

Comment:
nitpick: how about renaming `reorgEventsReader` to `reorgEventsNotifier` or `reorgNotifier`? I am thinking about matching already used naming pattern e.g. `newBlockNotifier` channel.

Reply:
cool, makes sense